### DISCUSSION TITLE
潜水员戴夫|豪华中文|V1.0.2.1429-周年特别版+全DLC+修改器|解压即撸

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1592] [其他]
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/2841140/capsule_616x353.jpg?t=1716451311)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/bundles/36551/4ayu7cik7tttcfji/capsule_616x353.jpg?t=1692149536)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/capsule_616x353.jpg?t=1692149536)
欢迎访问以神秘蓝洞为背景的海洋冒险游戏【潜水员戴夫 DAVE THE DIVER】。 白天在美丽的大海里打鱼，晚上经营寿司店赚钱。 伴随着个性强的同事们的故事展开的海底谜团。神秘又崭新的乐趣正在等着您！
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/header.jpg?t=1688092032)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/2492320/header.jpg?t=1688197590)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/extras/DR_Steam_Banner_F.png?t=1688092032)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/extras/DAVE_THE_DIVER_GIF_1.gif?t=1688092032)
探索神秘的蓝洞!
美丽的海底中充满了无数鱼种和巨大生物，对何时何地出现的海洋生物，是不是好奇呢？
避开威胁生物的袭击，获得新鲜的食材。
梦幻般的海洋生态和神秘的古代遗物。海里充满了谜团。
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/extras/DAVE_THE_DIVER_GIF_2.gif?t=1688092032)
只在夜间营业的寿司店
用亲自抓来的新鲜食材做出特别的寿司并销售吧！如果口碑传开，可能会有特别的客人突然来访。
经营寿司店筹集资金，可以探索更深的地方！
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/extras/DAVE_THE_DIVER_GIF_3.gif?t=1688092032)
在深海中存在的危险
当心！异常现象导致的巨大蓝洞里有未知生命体等着您的到来。随时充分强化您的装备。对抗巨大生物，对保护自己会有很大帮助的。
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1868140/ss_52b5730f94d6a6cfab5491faa125497f4e63209e.1920x1080.jpg?t=1688092032)

链接: https://pan.baidu.com/s/1Ox2XsbH0621Z6EWqQGy9SQ

提取码: q6aq

解压密码:laoquzhang.com

中文修改器[外站]:[点我](https://2468c.com/5844.html)

**Comment by Tang-Li-1** [comment]
**中国人自己的游戏网站**
